packages/cli/src/commands/replay/replay.command.ts

```
@@ -296,8 +294,9 @@ export const replayCommandHandler = async ({
       });
     } catch (error) {
       if (exitOnMismatch) {
-        throw error;
+        process.exit(1);
       }
+      throw error;
```

Note: this was a bug; it was swallowing screenshot diff failures.